bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-overlay/pan-zoom-draw-listener/component.jsx

```diff
@@ -160,7 +160,7 @@ export default class PanZoomDrawListener extends React.Component {
   }
 
   render() {
-    const baseName = Meteor.settings.public.app.basename;
+    const baseName = Meteor.settings.public.app.cdn || Meteor.settings.public.app.basename;
```

Comment:
The other ones use `cnd + basename`, but this one is `cdn || basename`. Is that intentional?

Reply:
It is intentional. If the the cdn value is empty we will just use the basename, if it is not it will prepend the basename with the cdn.

Reply:
I'm still confused. This pan-zoom-listener has:
```
const baseName = Meteor.settings.public.app.cdn || Meteor.settings.public.app.basename;
```
but pencil-draw-listener has:
```
const baseName = Meteor.settings.public.app.cdn + Meteor.settings.public.app.basename;
```

If I understand your comment correctly, they should all be `cdn + basename`. Using an OR won't prepend the basename with the CDN.

Reply:
Sorry. I forgot to update this one and the next. Going to fix them, we need + not |